geo-traits/src/polygon.rs

```diff
@@ -0,0 +1,64 @@
+use super::line_string::LineStringTrait;
+use geo_types::{CoordNum, LineString, Polygon};
+use std::iter::Cloned;
+use std::slice::Iter;
+
+pub trait PolygonTrait<'a>: Send + Sync {
```

Comment:
I wonder if we'll want to specify guarantees around winding order for implementors.

Reply:
I don't know... some stores might not have that information? I think GEOS objects don't have a known winding order?